subsys/power/device.c

```diff
@@ -64,6 +64,23 @@ int sys_pm_suspend_devices(void)
 	return 0;
 }
 
+int sys_pm_force_suspend_devices(void)
+{
+	for (int i = device_count - 1; i >= 0; i--) {
+		int idx = device_ordered_list[i];
+
+		device_retval[i] = device_set_power_state(&pm_device_list[idx],
+						DEVICE_PM_FORCE_SUSPEND_STATE);
```

Comment:
Not your fault, since the original code is not following it from the start. But indent of parameters should be below last '(' + 1 space (unless it can't fit in 80 chars, then it's more relaxed or then up to first parameter to be on its own line, and next ones right below).

the LOG_ERR below has same issue.

Reply:
OK will it.